holmes/core/tool_calling_llm.py

```diff
@@ -100,7 +100,7 @@ def call(
     ) -> LLMResult:
 
         tool_calls = []
-        tools = self.tool_executor.get_all_tools_openai_format()
+        tools = self.tool_executor.get_all_tools_openai_format(self.llm.model)
```

Comment:
maybe it makes sense to write a more generalized method to check model and if model gemini use  `get_gemini_format`, otherwise use `get_openapi_format` because we might have much more cases like this in the future ?